content/en/docs/tutorials/trust-policy.md

```diff
@@ -0,0 +1,265 @@
+            },
+            "trustStores": [ "ca:expired-example" ],
+            "trustedIdentities": [
+                "*"
+            ]
+        }
+    ]
+}
+```
+
+Use `notation verify` to verify the image using the *permissive* verification level and *expired-example* trust store and confirm it succeeds while logging the expired certificate.
+
+```console
+$ notation verify $IMAGE
+Warning: expiry was set to "logged" and failed with error: digital signature has expired on "Fri, 09 Dec 2022 16:30:07 -0600"
+Successfully verified signature for localhost:5000/net-monitor@sha256:1111111111111111111111111111111111111111111111111111111111111111
+```
+
+Update the trust policy to use the *missing-example* trust store.
+
+```json
+{
+    "version": "1.0",
+    "trustPolicies": [
+        {
+            "name": "trust-policy-example",
+            "registryScopes": [ "*" ],
+            "signatureVerification": {
+                "level" : "permissive" 
+            },
+            "trustStores": [ "ca:missing-example" ],
+            "trustedIdentities": [
+                "*"
+            ]
+        }
+    ]
+}
+```
+
+Use `notation verify` to verify the image using the *permissive* verification level and *missing-example* trust store and confirm it fails.
+
+```console
+$ notation notation verify $IMAGE
+Error: signature verification failed for all the signatures associated with localhost:5000/net-monitor@sha256:1111111111111111111111111111111111111111111111111111111111111111
+```
+
+This example fails because the *permissive* verification level does not allow missing certificates. When creating the *missing-example* certificate and trust store, the certificate was not used to sign the image.
+
+Update the trust store to use the *audit* verification level.
+
+```json
+{
+    "version": "1.0",
+    "trustPolicies": [
+        {
+            "name": "trust-policy-example",
+            "registryScopes": [ "*" ],
+            "signatureVerification": {
+                "level" : "audit" 
+            },
+            "trustStores": [ "ca:audit-example2" ],
+            "trustedIdentities": [
+                "*"
+            ]
+        }
+    ]
+}
+```
+
+Use `notation verify` to verify the image using the *audit* verification level and *audit-example2* trust store and confirm it succeeds while logging the missing certificate.
+
+```console
+$ notation verify $IMAGE
+Warning: authenticity was set to "logged" and failed with error: signature is not produced by a trusted signer
+Successfully verified signature for localhost:5000/net-monitor@sha256:1111111111111111111111111111111111111111111111111111111111111111
+```
+
+## Cleanup
+
+### Remove the notation configuration
+
+To remove local keys, self-signed certificates, and notation configurations, remove the notation configuration directory.
+  
+For linux:
+
+```console
+rm -r ${HOME}/.config/notation/
+```
+
+For macOS:
+
+```console
+rm -r ${HOME}/Library/Application\ Support/notation/
```

Comment:
Maybe we should add double quotes for the path since we have `\ ` in the path?  "${HOME}/Library/Application\ Support/notation/"

Reply:
the `\` is to escape the space